Source/Charts/Data/Interfaces/IChartDataSet.swift

```diff
@@ -10,6 +10,7 @@
 //
 
 import Foundation
+import CoreGraphics
```

Comment:
It's interesting in my Xcode, it never warns or complains missing CoreGraphics, though IChartDataSet refer CGPoint class